planner/core/point_get_plan.go

```diff
@@ -188,7 +188,7 @@ func tryPointGetPlan(ctx sessionctx.Context, selStmt *ast.SelectStmt) *PointGetP
 	handlePair := findPKHandle(tbl, pairs)
 	if handlePair.value.Kind() != types.KindNull {
 		if len(pairs) != 1 {
-			return nil
+			goto CheckUK
```

Comment:
Since we can merge the two if, why `goto` here?

Reply:
How about wrap 207-226 lines to a function, and call it here, to avoid `goto`

Reply:
Both OK, we should try the effect first.